helm/kong-app/templates/pdb.yaml

```diff
@@ -19,9 +19,7 @@ spec:
   {{- if .Values.podDisruptionBudget.maxUnavailable }}
   maxUnavailable: {{ .Values.podDisruptionBudget.maxUnavailable }}
   {{- end  }}
-  {{- if semverCompare ">=1.26.0-0" .Capabilities.KubeVersion.Version }}
```

Comment:
would you include a note in our Changelog that this breaks compatibility with v1.25?